extensions/amp-video-docking/validator-amp-video-docking.protoascii

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-video-docking
+  attr_lists: "common-extension-attrs"
+  extension_spec: {
+    name: "amp-video-docking"
```

Comment:
Should this extension depend on the `amp-video` extension as well?

Reply:
Eventually this feature will be available to more than one video player. There's no such thing as `oneof` for `requires_extension`.

Reply:
...although I realized we can set different specs for each element that supports it. Updated.